tests/test_apns.py

```diff
@@ -279,3 +279,93 @@ def test_apns_create_socket_empty_certificate(tmpdir):
 
     with pytest.raises(exceptions.APNSAuthError):
         apns.create_socket(TCP_HOST, TCP_PORT, str(certificate))
+
+
+class TestAuthToken(object):
```

Comment:
Tests should be defined as plain functions instead of using classes.

Reply:
This class is to group the relevant tests into the same block, it makes the context of (a group of) tests more intuitive. 

I am more than happy to adapt your convention if you strongly prefer it.

Reply:
I am purposely not using test classes.

Reply:
OK. I will adapt your convention.